src/libraries/Common/src/Interop/Windows/Kernel32/Interop.ReadConsoleInput.cs

```diff
@@ -16,7 +16,7 @@ internal struct KeyEventRecord
         internal short repeatCount;
```

Comment:
Unrelated nit: The name of this struct should be `KEY_EVENT_RECORD` to follow the coding conventions.